src/util/apiHelpers.ts

```diff
@@ -229,3 +228,18 @@ export const fetchWithAuthorization = async (
     },
   });
 };
+
+export async function fetchWithFeideAuthorization(
+  url: string,
+  forceAuth?: boolean,
+) {
+  if (forceAuth || !isAccessTokenValid()) {
+    //await renewAuth();
```

Comment:
Skal det skje noko her?

Reply:
Ja det denne gjorde var å prøve å logge inn på nytt, dersom man ikke har valid access token. Var noe jeg kopierte fra en annen funksjon.
Men tror det var noe problem med tokens mens jeg drev å testa, kan legge det inn igjen

Reply:
Så lenge du ikkje tvinges til å logge inn. Det var en feil i originalkoden til Lars som blei fjerna.

Reply:
Dette er for endepunkt som krever at man er logget inn, så er vel egentlig feil at den i det hele tatt blir kjørt om man ikke er forventet å være logget inn